etc/teams-for-linux.profile

```diff
@@ -0,0 +1,27 @@
+include disable-common.inc
+include disable-devel.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include globals.local
+
+caps.drop all
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+nou2f
+novideo
+protocol unix,inet,inet6,netlink
+seccomp
```

Comment:
What about `shell none` and `tracelog`?

Reply:
Tried adding both of these, and the program doesn't load with these defined (both).  Curious about what tracelog does, so I'll grep the source to learn about that.

Reply:
If it break, leave it. If you don't know what option XY do, you can read the source-code or read the manpages (`man firejail` and `man firejail-profile`) :wink:

Reply:
I was able to get shell none to work, but not tracelog.  Committed and pushed that part of the change.